scenes/quests/lore_quests/quest_001/2_ink_combat/ink_combat_round_1.tscn

```diff
@@ -109,7 +109,7 @@ position = Vector2(0, 65)
 position = Vector2(0, -25)
 collision_layer = 4
 script = ExtResource("5_lmnu5")
-scene_to_go_to = "uid://cgemeabsewy1f"
+scene_to_go_to = "uid://bj1lfa32hmtqi"
```

Comment:
I think this change is wrong. The current UID (`cge...`) is the correct one for round 2:

```
wjt@camille:threadbare (main)$ head -n1 scenes/quests/lore_quests/quest_001/2_ink_combat/ink_combat_round_2.tscn
[gd_scene load_steps=11 format=4 uid="uid://cgemeabsewy1f"]
```

`bj1lfa32hmtqi` does not appear in the source tree in `main`.